src/application/components/ProjectsList.tsx

```diff
@@ -0,0 +1,39 @@
+import { Fragment } from 'react';
+
+import { ProjectInfo } from '../../api/projectsAPI';
+import { emitEvent } from '../../events';
+import { cx } from '../../lib/cx';
+import { RefStudioEditorIcon } from './icons';
+
+export function ProjectsList({ projects }: { projects: ProjectInfo[] }) {
+  return (
+    <div className="flex flex-col items-stretch gap-4 overflow-y-hidden">
+      <h1>Recent Projects</h1>
+      <div className="flex flex-col items-stretch gap-2 overflow-y-scroll">
+        {projects.map((project, index) => (
+          <Fragment key={project.id}>
+            {index > 0 && <div className="h-[1px] shrink-0 bg-welcome-border" />}
+            <ProjectItem project={project} />
+          </Fragment>
+        ))}
+      </div>
+    </div>
+  );
+}
+function ProjectItem({ project }: { project: ProjectInfo }) {
+  return (
+    <div
+      className={cx(
+        'flex cursor-pointer items-center gap-2 p-2 pr-3',
+        'rounded-default hover:bg-btn-bg-side-bar-item-hover',
+        'text-btn-txt-side-bar-item-primary',
+      )}
+      onClick={() => emitEvent('refstudio://projects/open', { projectId: project.id })}
```

Comment:
Can you add an event to this component (and the parent) for clicks in the project, and then implement the emitEvent outside. This will allow me to reuse this in #447